programs/svm-spoke/src/state/refund_account.rs

```diff
@@ -0,0 +1,132 @@
+use anchor_lang::prelude::*;
```

Comment:
nit on this file naming: we call the instruction `refund_claims` and the instruction `refund_accounts`. we should think about it makes sense to re-name this state to `claim_account`? equally, for other files between instruction & state having a consistent name between the two.

Reply:
`RefundAccount` can hold either `TokenAccount` or `ClaimAccount` and `state/refund_account.rs` has methods for handling both, so would not want to name it as `claim_account`. Also the wrapped `RefundAccount` is used in `bundle` instruction while individual `ClaimAccount` is used in `refund_claims` instruction, so there is no 1-1 relationship between state and instructions like we have in other places.